tools/base/src/Base.ts

```diff
@@ -185,8 +187,14 @@ export function SpectrumMixin<T extends Constructor<ReactiveElement>>(
     return SpectrumMixinElement;
 }
 
+export type SpectrumConfig = {
+    downstate?: string[];
```

Comment:
I am not sure I follow why this needs to be configurable. If all the downstate controller does is update the value of some CSS variables, why does it need to be system-specific?
I would expect these CSS variables to not produce effect on non-supported systems.

Reply:
So this is supposed to be configurable because we wanted to cover up for any further delegates that we might have to add specific to the system... Downstate is just one example that we already know.

Reply:
Right, but by doing this we are introducing a JS-based feature toggling mechanism that, in this particular instance, could be avoided. Given that the CSS variables should inherently not affect unsupported systems, it seems unnecessary to add this extra layer of configuration.

I'm still not 100% convinced that adding extra code now in anticipation of unknown future features is the best approach, especially because it moves us away from CSS being the primary source of truth, which represents a significant shift in our mental model.